File: angular_workspace/pfiva-web/src/app/data-model/survey/SurveyResponseData.ts
import { Survey } from "./Survey";
import { Response } from "./Response";

export class SurveyResponseData {
	private survey: Survey;
	private responses: Response[];

	public get Survey(): Survey {
		return this.survey;
	}
	public set Survey(value: Survey) {
		this.survey = value;
	}
	public get Responses(): Response[] {
		return this.responses;
	}
	public set Responses(value: Response[]) {
		this.responses = value;
	}

}

File: workspace/data-model/src/main/java/de/pfiva/data/model/survey/SurveyResponseData.java
package de.pfiva.data.model.survey;

import java.util.Set;

public class SurveyResponseData {

	private Survey survey;
	private Set<Response> responses;

	public Survey getSurvey() {
		return survey;
	}

	public void setSurvey(Survey survey) {
		this.survey = survey;
	}

	public Set<Response> getResponses() {
		return responses;
	}

	public void setResponses(Set<Response> responses) {
		this.responses = responses;
	}

	@Override
	public String toString() {
		return "SurveyResponseData [survey=" + survey + ", responses=" + responses + "]";
	}
}
